lib/src/query_component.dart

```diff
@@ -116,11 +115,12 @@ class QueryComponent implements OnInit {
 
   /// Fetches revenue spent data using DBM reporting APIs,
   /// then return a map with <ioID, revenue> key-value pairs.
-  Future<Map<String, String>> _queryAndParseRevenueSpentData(
-      InsertionOrder_Budget_BudgetSegment_DateRange dateRange) async {
+  Future<Map<String, List<InsertionOrderDailySpend>>> _queryAndParseSpentData(
```

Comment:
is there an existing multimap data type you could use? quiver lib has some more advanced data types like multimaps

Reply:
wooo me likey